### ISSUE TITLE
Section "Install from source" of documentation is empty

### ISSUE THREAD

**Issue by davide84** [girder/slicer_package_manager#71]
It just tells you to 'cd' into the cloned folder. And then?

**Comment by davide84** [comment]
In python_client/README.rst I found the answer. I suggest it should be reported in the online documentation:

```
To install from the latest source, first obtain the source code::

    $ git clone https://github.com/girder/slicer_package_manager.git
    $ cd slicer_package_manager/python_client

Then install with::

    $ pip install .

or::

    $ pip install -e .

for development.
```


Also in python_client/README.rst it reports an undocumented possibility:

```
To install with pip::

    $ pip install slicer-package-manager
```

but then:

```
$ pip3 install slicer-package-manager
ERROR: Could not find a version that satisfies the requirement slicer-package-manager (from versions: none)
ERROR: No matching distribution found for slicer-package-manager
```

**Comment by jcfr** [comment]
Good point, I will update the GitHub README to reference https://slicer-package-manager.readthedocs.io/en/latest/index.html

**Comment by jcfr** [comment]
>  I suggest it should be reported in the online documentation:

It is referenced from https://slicer-package-manager.readthedocs.io/en/latest/user_guide.html#how-to-use-the-slicer-package-manager

**Comment by jcfr** [comment]
> It just tells you to 'cd' into the cloned folder. And then?

Could you reference here the relevant URL ?

**Comment by davide84** [comment]
Hi! Sorry for the delay.
The relevant URL is https://slicer-package-manager.readthedocs.io/en/latest/installation.html section "Install from source".
What I was expecting it is actually located in https://slicer-package-manager.readthedocs.io/en/latest/commands_shell.html.
Maybe the section "Install from source" that I consider empty could be renamed "Download the source" and be moved as a subsection of "Run via Docker". Right now it looks like it's an alternative to it.

A bit of context: I'm working with slicer and needed to download an extension for it - the file. At some point in my search I found a comment like "the current extension manager is on life support, we're moving to girder packages, checkout the new client".
I managed to pip install it but I ended up not using it, because I have no username/password nor an API key, and I downloaded the extension I needed in the old way.
